### PR TITLE
44 about durysta spacing

### LINKED ISSUE #44: "about durysta" section spacing
image

- [x] space on the top 
- [x] space on the bottom 
- [x] adjust content in the word doc
- [x] image size ? ( live seems a fraction smaller )

### PR DESCRIPTION
**Pull request by rootpea**
Fix #44

Test URLs:
- Before: https://main--durysta--hlxsites.hlx.page
- After: https://44-about-durysta-spacing--durysta--hlxsites.hlx.page

### CODE REVIEW

**Comment by dragoss-nc** [comment]
Hello,

Here's a list of some inconsistencies:
- Image is bigger on desktop
image
- The bullets from the list are black, while on live they have a brownish shade.
image
- On live there is a dedicated image that is loaded for mobile breakpoint; it is focused more on the lady, so less of the couch is visible.
image

File: styles/styles.css [hunk 1]
```diff
@@ -475,6 +483,11 @@ main .section-divider {
   box-sizing: border-box;
 }
 
+main p:has(.section-divider) {
```
Comment (dragoss-nc) [inline styles/styles.css:486]: While this is really nice, unfortunately the support for FF is hidden behind a flag. Is this acceptable? I mean the tradeoff is a bit of spacing and miss-alignment.

File: styles/styles.css [hunk 2]
```diff
@@ -509,3 +522,34 @@ main .site-map-description ul {
...
+main .about-durysta ul {
+  margin-top: 15px;
+  margin-bottom: 30px;
+}
+
+main .about-durysta .columns {
+  margin-top: 31px;
+}
+
+main .about-durysta p:has(strong) {
```
Comment (dragoss-nc) [inline styles/styles.css:535]: Same as above.

**Review by dragoss-nc — APPROVED** [review]

**Comment by rootpea** [comment]
> * On live there is a dedicated image that is loaded for mobile breakpoint; it is focused more on the lady, so less of the couch is visible.
> 
> image

Thanks, I moved this to a new issue https://github.com/hlxsites/durysta/issues/74

**Pull request merged**